R/mediate.R

```diff
@@ -6,42 +6,42 @@
 #'                                                                              
 #' @param x a \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}.
 #' 
-#' @param outcome A single character value indicating the colData variable used
+#' @param outcome \code{Character scalar}. Indicates the colData variable used
 #'   as outcome in the model.
 #' 
-#' @param treatment A single character value indicating the colData variable
+#' @param treatment \code{Character scalar}. Indicates the colData variable
 #'   used as treatment in the model.
 #'
-#' @param mediator A single character value indicating the colData variable used
-#'   as mediator in the model. (default: \code{mediator = NULL})
+#' @param mediator \code{Character scalar}. Indicates the colData variable used
+#'   as mediator in the model. (Default: \code{NULL})
 #'
-#' @param assay.type A single character value indicating the assay used for
-#'   feature-wise mediation analysis. (default: \code{assay.type = NULL})
+#' @param assay.type \code{Character scalar}. Specifies the assay used for
```

Comment:
In fact, the default value should be "counts". Also, the input check is not using the common .check_assay_present() function() --> can you

1. Change default NULL to "counts"
2.Replace line 240 with .check_assay_present()